Basic Rules.txt

```diff
@@ -350,6 +350,47 @@ Using a medikit:
 Medic-kit, non-medic user: works as if you are a medic with skill 30.
 Medic-kit, medic user: Works as normal, but add 30 to amount healed.
 
+
+==============================
+Death, Dying and Downed
+==============================
+Oh no! You have less than 1 health point left! What now!?!
+
+The good news is you are not* dead yet. You are bleeding out. 
+When your character falls below 1 hit point, you get 3 saving throws called Death 
+Saving Throws. These throws are not tied to any attribute. When you fail any three 
+times, you die. If you succeed, you do not lose one of these three Death 
+Saving Throws, but will still have to roll next round if you are not stabalized. 
+Once you have expended a saving throw, the next time you fall 
+below 1 health point your saving throws fall to 2. the next time, you have 1. you 
+can only replenish these saving throws by going a whole encounter without having 
+to use any. Each encounter that you go without expending a saving throw 
+replenishes one saving throw up to a maximum of three.
+
+Downed
+The state of having less than one Health Point is called being "downed." It is also
+sometimes called "bleeding out." They are equivalent. While a character is downed,
+they cannot move, cannot stand under their own power and fall prone if not 
+supported by other means. They cannot speak, but can mumble incomprehensively.

```

Comment:
>They cannot speak, but can mumble incomprehensively.

better as

>They cannot speak but can mumble in incomprehensibly.

or

>They cannot speak, but they can mumble incomprehensibly

...because "can mumble incomprehensively" is not a complete sentence so a comma isn't necessary, and "incomprehensively" isn't spelled correctly.

Reply:
spell checked.